test/bin/mdx-pp/expect/section-option/test_case.ml.expected

```diff
@@ -1,6 +1,6 @@
-#7 "test-case.md"
+#8 "test-case.md"
```

Comment:
I think we offset all locations by 1 line in mdx-pp now.

Reply:
fixed